packages/nodes-base/nodes/Code/Code.node.ts

```diff
@@ -107,16 +108,15 @@ export class Code implements INodeType {
 				context.item = context.$input.item;
 			}
 
-			if (language === 'python') {
-				context.printOverwrite = workflowMode === 'manual' ? this.sendMessageToUI : null;
-				return new PythonSandbox(context, code, index, this.helpers);
-			} else {
-				const sandbox = new JavaScriptSandbox(context, code, index, workflowMode, this.helpers);
-				if (workflowMode === 'manual') {
-					sandbox.vm.on('console.log', this.sendMessageToUI);
-				}
-				return sandbox;
-			}
+			const Sandbox = language === 'python' ? PythonSandbox : JavaScriptSandbox;
+			const sandbox = new Sandbox(context, code, index, this.helpers);
+			sandbox.on(
+				'output',
+				workflowMode === 'manual'
+					? this.sendMessageToUI
+					: (...args) => console.log(`[Node "${node?.name}"]`, ...args),
```

Comment:
Is it actually a good idea to console.log production executions? This can potentially pollute our logs. If we want to keep production logs it would be helpful to know the workflow (not only the node).

Reply:
I assumed that the `console: 'inherit'` forwarded production executions logs automatically to stdout. If that's not what it's doing, then we should definitely not log here either. I just wanted to make sure that this change was backward-compatible.
Will test now, and update accordingly if needed.

Reply:
Oh you are right, we previously did console.log in production. I guess than let's keep it.

Reply:
Just checked. we are already forwarding the console.log calls for production executions. However, the changes in this PR make the output in the backend consistent with the logs in the browser.
I'll update the production executions to log the workflow ids as well.